src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList-Create.cshtml

```diff
@@ -1,20 +1,24 @@
 @model ContentOptionsViewModel
-<div class="form-group">
+<div class="form-group d-inline-flex float-right mb-0">
     <div class="btn-group">
         @if (Model.CreatableTypes.Any())
         {
             @if (Model.CreatableTypes.Count == 1)
             {
-                <a class="btn btn-sm btn-success" href="@Url.RouteUrl(new { area = "OrchardCore.Contents", controller = "Admin", action = "Create", id = Model.CreatableTypes.First().Value, returnUrl = FullRequestPath })">@T["New {0}", Model.CreatableTypes.First().Text]</a>
+                <a class="btn btn-sm btn-secondary" href="@Url.RouteUrl(new { area = "OrchardCore.Contents", controller = "Admin", action = "Create", id = Model.CreatableTypes.First().Value, returnUrl = FullRequestPath })">@T["New {0}", Model.CreatableTypes.First().Text]</a>
```

Comment:
Why are you changing this button color?

Reply:
> Why are you changing this button color?

To make it suited with the filter color scheme

> No need to do 5 different PR's just change those that are easy to do for now in all other places you can find.

You 're right, after this PR, coz we cover all the cases, I can merge content types and content parts into a single PR, but as always Seb said make small manageable PRs, so it could be easy to review ;)